### PR TITLE
Remove extraneous quotation marks

### PR DESCRIPTION
**Pull request by NathanJang**

### CODE REVIEW

File: examples/7-multiple-external-providers.html [hunk 1]
```diff
@@ -163,7 +163,7 @@
             gapi.auth.authorize({
               client_id:        '694766332436-1g5bakjoo5flkfpv3t2mfsch9ghg7ggd.apps.googleusercontent.com',
               scope:            ['https://www.googleapis.com/auth/plus.me'],
-              'approval_prompt': 'force',
```
Comment (marcoow) [inline examples/7-multiple-external-providers.html]: would be cool if the values would be still aligned ;)

**Comment by NathanJang** [comment]
Done.

**Comment by NathanJang** [comment]
Sorry about that; I'm not used to aligning key-value pairs.

**Comment by marcoow** [comment]
thanks!

**Pull request merged**